src/TSTypeGen/TypeUtils.cs

```diff
@@ -62,10 +62,14 @@ public static string GetFullNameWithGenericArguments(Type type)
 
         public static List<PropertyInfo> GetRelevantProperties(Type type)
         {
-            return type
-                .GetProperties(BindingFlags.Instance | BindingFlags.GetProperty | BindingFlags.Public | BindingFlags.DeclaredOnly)
-                .Where(p => p.GetIndexParameters().Length == 0)
+            var props = type
```

Comment:
whats the reasoning behind these changes? did you just do them to easier debug or do you think they add value in readability?